cc_licenses/templates/includes/view_legal_code_link_plain_text.html

```diff
@@ -1,5 +1,5 @@
 {% load i18n %}
 
 <div id="legal-code-plain-text" style="font-weight: bold;" class="padding-vertical-normal" >
-  <p class="body-big"><a href="#" class="link">{% trans "View Legal Code as plain text" %}</a></p>
+  <p class="body-big"><a href="{{ request.get_full_path }}.txt" class="link" download="{{ legalcode.fat_code }}.txt">{% trans "View Legal Code as plain text" %}</a></p>
```

Comment:
I thought in our last status call, the intention was that the plain text link just display the plain text in the browser.  E.g. serve the plain text with text/plain content type. If so, we wouldn't want the download attribute.

Reply:
Am I right in thinking this will only work from the published static site, and not the live site?

It'd be nice if it'd work both places. Could we make a Django view for the plain text files, that generates them on demand using pandoc, then add the view to django-distill?  I think we could create an HttpResponse, then pass it to subprocess as stdout so pandoc's output will go into the response.

Reply:
That is correct. I'll work on adding that view.

Reply:
Probably don't need pandocs for the above.